include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
It seemed the previous code was attempting to do this, but there were two problems:
1. It wouldn't compile on Windows due to ambiguity.
2. On Unix it would compile, but the wrong specialization would be used, leading to a crash.

I don't know if MS of clang was correct in respect to the standard.

So at this point the include is a necessity to print std::vector\<bool\>. As vector can be hidden inside an inline namespace forward declaration will be ugly.

The best I can offer in regards to speed/memory is that I have a commit that doesn't even bother RuntimePrintValue.h for builtin types.